docker/build.sh

```diff
@@ -40,6 +40,7 @@ set -e
 TARGET="${TARGET:-leonardo}"
 DB_CONTAINER="leonardo-mysql"
 GIT_BRANCH="${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}"
+REMOTE=$(if [[ ${GIT_BRANCH} == "update/"* ]]; then echo "scalaSteward"; else echo "origin"; fi)
```

Comment:
I guess this would break non-scalasteward branches that start with `update/`. That's probably fine, just noting it :)

Reply:
yea...this is assuming no one names their branch `update/xxx`